site/content/community/_index.md

```diff
@@ -14,9 +14,9 @@ You can follow the work we do, see our milestones, and our backlog on our [GitHu
 * Follow us on Twitter at [@projectvelero](https://twitter.com/projectvelero)
 * Join our Kubernetes Slack channel and talk to over 800 other community members: [#velero](https://kubernetes.slack.com/messages/velero)
 * Join our [Google Group](https://groups.google.com/forum/#!forum/projectvelero) to get updates on the project and invites to community meetings.
-* Join the Velero community meetings:
-  * 1st and 3rd Tuesday at 12PM ET / 9AM PT ([Convert to your time zone](http://www.thetimezoneconverter.com/?t=09:00&tz=PT%20%28Pacific%20Time%29)) - [Zoom link](https://vmware.zoom.us/j/551441444?pwd=dHJyMWZtdHFPWWFJaTh5TnFuYWMvZz09)
-  * 2nd and 4th Tuesday at 8am China Standard Time / 8pm ET / 5pm PT ([Convert to your time zone](http://www.thetimezoneconverter.com/?t=17:00&tz=PT%20%28Pacific%20Time%29)) - [Zoom link](https://vmware.zoom.us/j/551441444?pwd=dHJyMWZtdHFPWWFJaTh5TnFuYWMvZz09)
+* Join the Velero community meetings  - [Zoom link](https://vmware.zoom.us/j/551441444?pwd=dHJyMWZtdHFPWWFJaTh5TnFuYWMvZz09):
+  * 1st and 3rd Tuesday at 12PM ET / 9AM PT ([Convert to your time zone](https://dateful.com/convert/est-edt-eastern-time?t=12pm))
+  * 2nd and 4th Wednesday at 8am China Standard Time / Tuesday ET ([Convert to your time zone](https://dateful.com/convert/beijing-china?t=8am))
```

Comment:
Looks like you're missing the Tuesday times for ET/PT

Reply:
Also, we might as well deal with the daylight savings/standard time issue here. Instead of "8pm ET/ 5pm PT", maybe we should say "7pm EST (8pm EDT) / 4 pm PST (5pm PDT)" -- since the 2nd/4th week meetings are based on China Standard time which doesn't switch for daylight savings.

Reply:
```suggestion
  * 2nd and 4th Wednesday at 8am China Standard Time / Tuesday 7pm EST (8pm EDT) / Tuesday 4 pm PST (5pm PDT) ([Convert to your time zone](https://dateful.com/convert/beijing-china?t=8am))
```